File: 대소문자변환.java
import java.util.*;

class Main {
  public String solution(String str) {
  	char[] array = str.toCharArray();
    StringBuilder sb = new StringBuilder();
    
    for(char a: array) {
    	if(Character.isUpperCase(a)) {
           sb.append(Character.toLowerCase(a));
        }
        else {
            sb.append(Character.toUpperCase(a));
        }
    }
    return sb.toString();
  }
  
  public static void main(String[] args){
    Main T = new Main();
    Scanner in=new Scanner(System.in);
    String str = in.next();

    System.out.println(T.solution(str));
    
  }
}

File: 대소문자변환.kt
import java.util.*

class Main {
    fun solution(str: String): String {
        val array = str.toCharArray()
        val sb = StringBuilder()

        for(a in array) {
            if(a.isUpperCase()) sb.append(a.toLowerCase())
            else sb.append(a.toUpperCase())
        }
        return sb.toString()
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val T = Main()
            
            val str = "StudY"
            println(T.solution(str))
        }
    }
}